package.json

```diff
@@ -45,7 +45,7 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "^1.1.1",
-    "@edx/frontend-build": "12.8.38",
+    "@edx/frontend-build": "13.0.14",
```

Comment:
Same as with `paragon`, `frontend-build` has also moved to the `openedx` scope (`@openedx/frontend-build`), Version `13.0.15` there should be the same as this.